monai/deploy/operators/dicom_data_loader_operator.py

```diff
@@ -249,7 +249,14 @@ def populate_series_attributes(self, series, sop_instance):
             pass
 
         try:
-            pixel_spacing_de = sop_instance[0x0028, 0x0030]
+            tag_pixel_spacing = 'PixelSpacing'    # tag (0x0028, 0x0030)
+            tag_imager_pixel_spacing = 'ImagerPixelSpacing'    # tag (0x0018, 0x1164)
+            if tag_pixel_spacing in sop_instance: pixel_spacing_de = sop_instance[tag_pixel_spacing]
```

Comment:
Code itself looks good to me, though the format checking fails and recommends "auto style fixes: ./run check --autofix", please see the build/test step, https://github.com/Project-MONAI/monai-deploy-app-sdk/runs/5761205272?check_suite_focus=true

Reply:
Thanks.